packages/eva/mapping.json

```diff
@@ -545,74 +545,68 @@
         "outline": {
           "mapping": {
             "state": {
+              "active": {
+                "backgroundColor": "color-primary-500-transparent-16"
+              },
               "disabled": {
-                "backgroundColor": "background-basic-color-2"
+                "backgroundColor": "color-basic-600-transparent-16",
+                "borderColor": "color-basic-600-transparent-24"
```

Comment:
Here I have the opposite question to the one above. Will it affect React Native?